.github/workflows/update-snapshots.yml

```diff
@@ -5,26 +5,6 @@ on:
 
 jobs:
 
-  docker-build-e2e:
-    runs-on: ubuntu-20.04
-    steps:
-      - name: Fail if branch is main
-        if: ${{ github.ref == 'refs/heads/main' }}
-        run: |
-          echo "This workflow should not be triggered with workflow_dispatch on main"
-          exit 1
-
-      - name: Delete unnecessary tools folder to prevent "out of space" error
-        run: rm -rf /opt/hostedtoolcache
-
```

Comment:
the 2 steps above are deprecated? if yes, why?

Reply:
what about the two steps above? `Fail if branch is main` and `Delete unnecessary...`

Reply:
`Delete unnecessary...`  - itis now a part of `docker-build-e2e`, so we don't need to have it here.
`Fail if branch is main` - there is the same check in `update_snapshots` job (a bit below in this file), so the `update-snapshots` is still disabled for `main` as it was before

Reply:
perfect! thank you

Reply:
With this particular PR, I don't touch the screenshot updating part. IMO we first need to agree on the structure re. how and when we build e2e docker image, then it will be clear how we introduce `update_snapshots` workflow into the process. I added this step as the next one to tackle after we complete the current PR.